glide.lock

```diff
@@ -131,8 +131,6 @@ imports:
   - tstrings
   - tsync
   - ttesting
-- name: github.com/vjeantet/grok
```

Comment:
looks like you did not update the glide.lock properly after the reintegration.
try an update by `glide update github.com/vjeantet/grok --strip-vendor`